backend/tests/test_b055_tenant_propagation.py

```
@@ -0,0 +1,84 @@
+    result = attribution.recompute_window(
+        _DummyTask("tenant-prop-attribution"),
+        tenant_id=tenant_id,
+        window_start=window_start,
+        window_end=window_end,
+        correlation_id=correlation_id,
+        model_version="1.0.0",
+    )
+
+    assert seen.get("value") == str(tenant_id)
+
+    async def _cleanup():
```

Ruff (0.14.11)

Missing return type annotation for private function `_cleanup`

Add return type annotation: `None`

(ANN202)